types/shapediver/appbuildertypecheck.ts

```diff
@@ -141,6 +141,15 @@ const IAppBuilderWidgetPropsImageSchema = z.object({
 	anchor: z.string().optional(),
 	alt: z.string().optional(),
 	target: z.string().default("_blank"),
+	isSvg: z.boolean().optional(),
+	maw: z.union([
+		z.number().optional(),
+		z.string().optional(),
+	]),
+	mah: z.union([
+		z.number().optional(),
+		z.string().optional(),
+	]),
```

Comment:
@LemToUp let's drop maw and mah from here again

Reply:
Dropped